.github/workflows/build.yml

```diff
@@ -20,6 +20,9 @@ jobs:
           - variant: vmware-dev
```

Comment:
For lack of a better place to comment - we'll need the new variant in the variant list in the top-level README.md.